android/src/main/kotlin/com/gojuno/commander/android/Adb.kt

```diff
@@ -22,7 +22,10 @@ private val buildTools: String? by lazy {
 }
 val aapt: String by lazy { buildTools?.let { "$buildTools/aapt" } ?: "" }
 
-fun connectedAdbDevices(): Observable<Set<AdbDevice>> = process(listOf(adb, "devices"), unbufferedOutput = true)
+fun connectedAdbDevices(unbufferedOutput: Boolean = true): Observable<Set<AdbDevice>> = process(
+        listOf(adb, "devices"),
+        unbufferedOutput = unbufferedOutput
```

Comment:
Why should user of adb function care about that though?

Reply:
Well, he shouldn't, and the default value is still true.

Reply:
Maybe we can use `false` here, too, need to test it more on other OSes. Will check it.

Reply:
ook